### ISSUE TITLE
[3.12.9] "add link and linked to media object"

### ISSUE THREAD

**Issue by CannotTouch** [datawhores/OF-Scraper#471]
Hi,
in the note of the release https://github.com/datawhores/OF-Scraper/releases/tag/3.12.9 you insert the
"add link and linked to media object".
What is that? :D I cannot find any reference to it in the documentation.

Thanks for support

**Comment by datawhores** [comment]
Yeah that was more of a change in the code base, it would not be in the documentation
